python/ray/autoscaler/v2/sdk.py

```diff
@@ -34,18 +32,7 @@ def request_cluster_resources(to_request: List[dict], timeout: int = 10):
         timeout: Timeout in seconds for the request to be timeout
 
     """
+    if not ray.is_initialized():
```

Comment:
hmm why is this necessary?

Reply:
so we can get the GCSClient that's initialized in the worker

Reply:
hmm actually i think the caller already checks it.